modules/nw-sriov-installing-operator.adoc

```diff
@@ -148,5 +148,10 @@ If the operator does not appear as installed, to troubleshoot further:
 
 +
 * Inspect the *Operator Subscriptions* and *Install Plans* tabs for any failure or errors under *Status*.
-* Navigate to the *Workloads* -> *Pods* page and check the logs for pods in the
-`openshift-sriov-network-operator` project.
+* Navigate to the *Workloads* -> *Pods* page and check the logs for pods in the `openshift-sriov-network-operator` project.
+* Check the namespace of the YAML file and if the annotation is missing, you can add the annotation `workload.openshift.io/allowed=management` to the Operator namespace with the following command:
++
+[source,terminal]
+----
+$ oc annotate ns/openshift-sriov-network-operator workload.openshift.io/allowed=management
```

Comment:
I think this only needed for SNO cluster.  if so, we need reference with `SNO` cc  yliu1@redhat.com